python_modules/dagster/dagster/_core/definitions/partition.py

```diff
@@ -325,7 +325,12 @@ def ml_model_for_each_ocean():
     """
 
     def __init__(self, partition_keys: Sequence[str]):
```

Comment:
```python
def takes_sequence(seq: Sequence[str]) -> str:
    return seq[0]

print(takes_sequence("foo")) # prints f
```

oh noes 🙈

didn't know about this pitfall

Reply:
```
>>> [f for f in [f for f in [f for f in [f for f in [f for f in [f for f in "f"]]]]]]
['f']
```